ocs_ci/helpers/odf_cephfs_snap.py

```
@@ -244,3 +252,110 @@ def delete_volumesnaps_volumesnapcontents(snap_list_names):
+    for snap_entries in TimeoutSampler(
+        timeout=timeout,
+        sleep=sleep,
+        func=get_cephfs_snap_entries,
+        snap_runner=snap_runner,
+    ):
+        entry = next(
+            (e for e in snap_entries if e["snapshot"] == ceph_snap_name),
+            None,
+        )
+        assert entry is not None, (
+            f"Snapshot '{ceph_snap_name}' disappeared after " f"failed delete attempt"
```

Ruff (0.15.18)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)